### PR TITLE
Update log to show proper values

### PR DESCRIPTION
**Pull request by nickconnorsmoov**
# Changes
Changes to log to display values properly. 

# Why Are Changes Being Made

Note:
It was considered moving some of the logic to a function but it was decided that this approach was fine and helped increase readability.

### CODE REVIEW

File: pkg/response/match/matcher.go [hunk 1]
```diff
@@ -34,8 +34,46 @@ func (m Matcher) FindAction(ed *ach.EntryDetail) *service.Action {
...
+			returnCode = action.Return.Code
+		}
+
+		if matcher.Amount != nil {
+			amount = matcher.Amount.Value
+		}
+
+		m.debugLog(
+			fmt.Sprintf(
+				"attempting matcher resp[%d]= AccountNumber: %s, Amount: %d, EntryType: %s, IndividualName: %s, RoutingNumber: %s, TraceNumber: %s, CopyPath: %s, CorrectionCode: %s, CorrectionData: %s, ReturnCode: %s",
```
Comment (adamdecaf) [inline pkg/response/match/matcher.go:65]: Could we switch this to using structured logging?

```go
if m.Debug {
  m.logger.With(log.Field{
    "account_number": log.String(matcher.AccountNumber),
    // ...
  })
}
```

Comment (nickconnorsmoov) [reply pkg/response/match/matcher.go:65]: I discussed that with @atonks2. We decided that not introducing too many internal "moov-isms" into the open source was probably the correct move.

Comment (atonks2) [reply pkg/response/match/matcher.go:65]: The irony is I thought if you saw us suddenly using our moovfinancial logging conventions here, @adamdecaf, you would say it doesn't fit our OSS style/conventions 😂 .

In light of Adam's comment, please do use our normal, structured logging here @nickconnorsmoov. I Led you astray.

Comment (adamdecaf) [reply pkg/response/match/matcher.go:65]: After seeing this PR I'm planning to switch the project over to structured logging going forward lol.

**Review by atonks2 — APPROVED** [review]

File: pkg/response/match/matcher.go [hunk 2]
```diff
@@ -34,9 +34,47 @@ func (m Matcher) FindAction(ed *ach.EntryDetail) *service.Action {
...
+				"amount":          log.Int(amount),
+				"entry_type":      log.String(string(matcher.EntryType)),
+				"individual_name": log.String(matcher.IndividualName),
+				"routing_number":  log.String(matcher.RoutingNumber),
+				"trace_number":    log.String(matcher.TraceNumber),
+				"copy_path":       log.String(copyPath),
+				"correction_code": log.String(correctionCode),
+				"correction_data": log.String(correctionData),
+				"return_code":     log.String(returnCode),
+			})
```
Comment (adamdecaf) [inline pkg/response/match/matcher.go:76]: Without calling `.Log` / `.Logf` there won't be an log emitted. 

```suggestion
			}).Log("attempting matcher")
```

**Pull request merged**